maps/clarion.dmm

```diff
@@ -38379,7 +38373,7 @@
 	},
 /obj/table/reinforced/bar/auto,
 /obj/machinery/glass_recycler/bar{
-	g_amt = 30
+	glass_amt = 20
```

Comment:
Rather than a map override, if the recyclers having glass at roundstart is intended why not change the default value?

Reply:
Good question. At the time I did this because each station had a different amount of glass available to the tender; I tried to keep the amount of glass the same per map just to cut down on the number of changes made to the game. On closer inspection, it's a bit ridiculous. I'll remove that from this PR and atomize the change